spec/models/usage_spec.rb

```diff
@@ -43,4 +43,49 @@
       end
     end
   end
+
+  describe 'number days' do
+    context 'start and end date are same' do
+      it 'returns 1' do
+        usage = FactoryGirl.create(
+          :usage,
+          start_date: "02/15/2015",
+          end_date: "02/15/2015")
+        expect(usage.number_days).to eq 1
+      end
+    end
+    context 'start and end date are different' do
+      it 'returns 5' do
+        usage = FactoryGirl.create(
+          :usage,
+          start_date: "02/15/2015",
+          end_date: "02/20/2015"
+        )
+        expect(usage.number_days).to eq 5
+      end
+    end
+  end
+
+  describe 'display_time_period' do
+    context 'shows time period' do
+      it 'returns 02/15/2015 - 03/15/2015' do
+        usage = FactoryGirl.create(
+          :usage,
+          start_date: "02/15/2015",
+          end_date: "03/15/2015"
+        )
+        expect(usage.display_time_period).to eq "02/15/2015 - 03/15/2015"
+      end
+    end
+  end
+
+  describe 'display_source_info_for_usage' do
+    context 'shows string of source info' do
+      it 'returns 10 gallons Heating Oil' do
+        source = FactoryGirl.create(:carbon_source, name: "Heating Oil", conversion_factor: 2)
+        usage = FactoryGirl.create(:usage, amount_used: 10, units: "gallons", carbon_source: source)
+        expect(usage.display_source_info_for_usage).to eq  "10.0 gallons Heating Oil"
```

Comment:
Line is too long. [85/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.